starport/templates/ibc/packet/messages/x/{{moduleName}}/types/messages_{{packetName}}_test.go.plush

```diff
@@ -0,0 +1,39 @@
+package types
+
+import (
+	"testing"
+
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	"github.com/stretchr/testify/require"
+)
+
+func TestMsgSend<%= packetName.UpperCamel %>_ValidateBasic(t *testing.T) {
```

Comment:
This is something we omitted in the original implementation, but we should also consider the message invalid if 
```
msg.port == ""
msg.channelID == ""
msg.timeoutTimestamp == 0
```
Maybe we can add this change in this PR

Reply:
Should we create a custom error for each fail validation?

eg:
```go
	if msg.Port == "" {
		return sdkerrors.Wrap(types.ErrInvalidPort, "invalid packet port")
	}
```

Reply:
We can use `sdkerrors.ErrInvalidRequest` since this is purely related to the content of the message

```
if msg.Port == "" {
	return sdkerrors.Wrap(sdkerrors.ErrInvalidRequest, "invalid packet port")
}
```